content/block_docs.md

```diff
@@ -35,16 +36,24 @@ hash: String
 ```
 
 ## Coinbase
-The proposer of the block is identified by the first transaction in the `transaction_list`.
-This transaction is called the *coinbase* and will get awarded the block mining reward for their work.
+The proposer of the block is identified by the source of the first transaction in `transaction_list`.
+This transaction is called the *coinbase*, and its source will get awarded by the block mining reward for their work.
+The amount of the reward is determined by `block_reward` field of [`/config`](/config).
 
-> Place one of your own transactions as the first item in `transaction_list`
+> Place one of your own transactions as the first item in `transaction_list`.
+> Otherwise, someone else will receive the reward!
 
 # Mining
 The _mining_ process for the hash involves;
-- Creating a temporary JSON object with `transaction_list`, `timestamp` and `nonce` values
+- Creating a temporary JSON object with `transaction_list`, `nonce`, and `timestamp` values
 - Serializing it
+    - **NOTE:** Serialized JSON must comply with the rules explained in hash section of [transaction](@/transaction_docs.md) page.
+    - The order of keys should be as follows: `transaction_list`, `nonce`, `timestamp`.
```

Comment:
An example like the one under Transaction#Hash might be better, your call

Reply:
Will do